tools/phenomenal/nmr/batman.xml

```diff
@@ -3,67 +3,32 @@
      <requirements>
        <container type="docker">docker-registry.phenomenal-h2020.eu/phnmnl/batman</container>
      </requirements>
-     <command><![CDATA[
-runBATMAN.R -i ${nmr_spectra_input}
-            -o \$PWD
-#if str( $options_file ) != None:
-            -p ${options_file}
-#end if
-#if str( $metabolite_template ) != None:
-            -u ${metabolite_template}
-#end if
-#if str( $target_metabolites ) != None:
-            -l ${target_metabolites}
-#end if
+     <command><![CDATA[     
+     runBATMAN.R -i ${nmr_spectra_input} -o \$PWD -p ${options_file} -u ${metabolite_template} -l ${target_metabolites}
```

Comment:
We have agreed with Jianliang to move away from the optional command provided that documentation is added on how we are supposed to create those files, so this is fine.